src/js/components/Video/Video.js

```diff
@@ -182,7 +182,11 @@ const Video = forwardRef(
           const scrubberRect = scrubberRef.current.getBoundingClientRect();
           const percent =
             (event.clientX - scrubberRect.left) / scrubberRect.width;
-          videoRef.current.currentTime = duration * percent;
+          if (!duration)
```

Comment:
The use of curly braces is a bit inconsistent in this if/else.

Reply:
done.